Javid_Alizada/05.Ansible_run/group_vars/sa_all.yml

```diff
@@ -0,0 +1,3 @@
+#sa_user_name: javid_alizada
+sa_comment: Javid Alizada
+sa_user_pass: $1$I2iQh1ir$I8E8PvBei1.Lzm6Vhf3Rf/
```

Comment:
could you please use vault for that

Reply:
Done Please check . 
Added Vault pass. with  "ansible-vault encrypt group_vars/sa_all.yml" . if you want i can send you my vault pass.
Thanks